.github/workflows/lint.yaml

```diff
@@ -29,13 +30,15 @@ jobs:
           key: ${{ runner.os }}-lerna-${{ hashFiles('**/yarn.lock') }}
 
       - name: Install dependencies
+        if: steps.node-modules-cache.outputs.cache-hit != 'true'
         run: |
-          yarn
+          yarn --prefer-offline
+          node_modules/.bin/lerna bootstrap
       - name: Lint & Fix
         run: |
-          git config --global user.email "noreply@hoprnet.org"
-          git config --global user.name "HOPR Linting robot"
+          git config user.name github-actions
+          git config user.email github-actions@github.com
           yarn lint:fix
           git commit -a -m "Fix linting" || true
-          git pull origin ${{ github.ref }}
+          git pull --quiet origin ${{ github.ref }}
```

Comment:
why?

Reply:
it spams the lint action with all changed branches such that it is hard to see whether there was a commit or not